manage_breast_screening/mammograms/jinja2/mammograms/special_appointments/provide_special_appointment_details.jinja

```diff
@@ -0,0 +1,104 @@
+{% extends 'layout-app.jinja' %}
+{% from 'warning-callout/macro.jinja' import warningCallout %}
+{% from 'checkboxes/macro.jinja' import checkboxes %}
+{% from 'textarea/macro.jinja' import textarea %}
+{% from 'back-link/macro.jinja' import backLink %}
+{% from 'button/macro.jinja' import button %}
+{% from 'django_form_helpers.jinja' import app_radios, form_error_summary %}
+
+{% block beforeContent %}
+  {{ backLink(back_link_params) }}
+{% endblock beforeContent %}
+
+{% block messages %}
+  {{ form_error_summary(form) }}
+{% endblock %}
+
+{% block page_content %}
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-two-thirds">
+
+    <h1 class="nhsuk-heading-l">
+      <span class="nhsuk-caption-l">
+        {{ caption }}
+      </span>
+      {{ heading }}
+    </h1>
+
+    {% macro details_textarea(form, support_reasons_value) %}
+      {% set details_field_name = support_reasons_value.lower() + "_details" %}
+      {% set details_field = form[details_field_name] %}
+      {% set details_field_id = details_field.auto_id %}
+      {% set details_field_value = details_field.value() %}
+      {% set textarea_params = {
+        "label": {
+          "text": "Describe support required"
+        },
+        "id": details_field_id,
+        "name": details_field_name,
+        "value": details_field_value,
+        "hint": {
+          "text": hint
+        } if hint
+      } %}
+      {% if details_field.errors %}
+        {% set error_params = {
+          "errorMessage": {
+            "text": form[details_field_name].errors | first
+          }
+        } %}
+        {% set textarea_params = dict(textarea_params, **error_params) %}
+      {% endif %}
+      {{ textarea(textarea_params) }}
+    {% endmacro %}
+
+    <form action="{{request.path}}" method="POST">
```

Comment:
Need to add `novalidate` to all forms to stop browser HTML5 validation kicking in

```suggestion
    <form action="{{request.path}}" method="POST" novalidate>
```

Reply:
Ah thanks for flagging! All the more reason to extract this into a base template / helper